src/features/smartSearch/components/filters/AllInSubOrg/index.tsx

```diff
@@ -0,0 +1,223 @@
+import { FC, useEffect, useState } from 'react';
+import { Chip, MenuItem } from '@mui/material';
+
+import FilterForm from '../../FilterForm';
+import {
+  AllInSuborgFilterConfig,
+  NewSmartSearchFilter,
+  OPERATION,
+  SmartSearchFilterWithId,
+  ZetkinSmartSearchFilter,
+} from '../../types';
+import StyledSelect from '../../inputs/StyledSelect';
+import { Msg } from 'core/i18n';
+import messageIds from 'features/smartSearch/l10n/messageIds';
+import useSubOrganizations from 'features/organizations/hooks/useSubOrganizations';
+import { useNumericRouteParams } from 'core/hooks';
+import useSmartSearchFilter from 'features/smartSearch/hooks/useSmartSearchFilter';
+import StyledItemSelect from '../../inputs/StyledItemSelect';
+
+type Props = {
+  filter:
+    | SmartSearchFilterWithId<AllInSuborgFilterConfig>
+    | NewSmartSearchFilter;
+  onCancel: () => void;
+  onSubmit: (
+    filter:
+      | SmartSearchFilterWithId<AllInSuborgFilterConfig>
+      | ZetkinSmartSearchFilter<AllInSuborgFilterConfig>
+  ) => void;
+};
+
+const AllInSuborg: FC<Props> = ({
+  filter: initialFilter,
+  onSubmit,
+  onCancel,
+}) => {
+  const { orgId } = useNumericRouteParams();
+  const suborgs = useSubOrganizations(orgId).data || [];
+  const [scope, setScope] = useState<'any' | 'single' | 'multiple'>('any');
+  const [selectedSuborgIds, setSelectedSuborgIds] = useState<number[]>([]);
```

Comment:
These are defaulting to `'any'` and `[]`, but they need to default to whatever is passed in the `filter` property.

Right now, I believe that this is what's causing a bug I found when I try to edit an instance of this filter. Steps to reproduce:

1. Add a suborg filter
2. Change the scope to anything except "any"
3. Select a sub-organization
4. Save the filter
5. Back in the overview, click the pencil to edit the filter again

Expected results: I expect to see the config I just made represented in the interface.
Actual results: The interface is in it's default state.